test/socket/test_socket.rb

```diff
@@ -771,13 +771,10 @@ def test_udp_recvmsg_truncation
   end
 
   def test_resolurion_error_error_code
-    # https://rubyci.s3.amazonaws.com/freebsd12/ruby-master/log/20231130T103002Z.fail.html.gz
-    omit if /freebsd/ =~ RUBY_PLATFORM
-
     begin
-      Socket.getaddrinfo("www.kame.net", 80, "AF_UNIX")
+      Socket.getaddrinfo("example.com", 80, "AF_UNIX")
     rescue => e
-      assert_equal(Socket::EAI_FAMILY, e.error_code)
+      assert_equal([Socket::EAI_FAMILY, Socket::EAI_FAIL].include?(e.error_code), true)
```

Comment:
It's better to use `assert_include`. 

https://github.com/test-unit/test-unit/blob/bdd1668e1e3c0875fd1b3a59df8f6ec6472014a1/lib/test/unit/assertions.rb#L1671